packages/core/android/src/main/kotlin/com/datadog/reactnative/DdRumImplementation.kt

```diff
@@ -333,6 +335,39 @@ class DdRumImplementation(private val datadog: DatadogWrapper = DatadogSDKWrappe
         }
     }
 
+
+    @ExperimentalRumApi
+    fun startFeatureOperation(name: String, operationKey: String? = null, attributes: ReadableMap, promise: Promise) {
```

Comment:
I think detekt is complaining about the missing documentation on these new functions.